capabilities/istio/.github/zarf-runner/deploy/zarf.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ZarfPackageConfig
+metadata:
+  name: farmer-pickles
+  url: https://welovebob.com/characters/farmer_pickles.htm
+  image: https://welovebob.com/Images/characters/farmer_pickles.jpg
+  description: "A zarf package to deploy things"
+
+components:
+  - name: deploy
+    required: true
+    actions:
+      onCreate:
+        before:
+          - description: Deploy the cluster bundle
+            cmd: |
+                ARCH=$(uname -m)
+                [ "$ARCH" = "x86_64" ] && ARCH="amd64"
+                ./uds deploy oci://ghcr.io/zachariahmiller/k3d-common:0.1.0-$ARCH --no-progress --confirm
+            dir: ../build
+          - description: Deploy the istio
+            cmd: |
+                ARCH=$(uname -m)
+                [ "$ARCH" = "x86_64" ] && ARCH="amd64"
+                ./zarf p d ../build/zarf-package-uds-capability-istio-$ARCH-1.19.3.tar.zst  --no-progress --confirm #${COMPONENTS}          
```

Comment:
Should this be `../build/zarf-package-uds-capability-istio-$ARCH-*.tar.zst` to ensure we don't have to keep it up to date with the built version?

Reply:
If that will actually work I think you are correct. Fwiw I had tried that and thought it wasn't working but maybe I was just being dumb.